export/src/export.js

```diff
@@ -100,7 +100,7 @@ module.exports = {
       }
       const cleanedVersion = version._doc
 
-      exportHTML({ bib: cleanedVersion.bib, yaml: cleanedVersion.yaml, md: cleanedVersion.md, id: cleanedVersion._id, title: article._doc.title }, res, req)
+      exportHTML({ bib: req.params.id.bib, yaml: req.params.id.yaml, md: req.params.id.md, id: req.params.id._id, title: article._doc.title }, res, req)
```

Comment:
Le BibTeX, le YAML et le Markdown proviennent toujours de la dernière version de l'article, il faut donc conserver `cleanedVersion.bib`, `cleanedVersion.yaml` et `cleanedVersion.md`.

`req.params.id` contient l'identifiant de l'article (chaîne de caractères), il faut donc utiliser `id: req.params.id` et non `id: id: req.params.id._id`:


```suggestion
      exportHTML({ bib: cleanedVersion.bib, yaml: cleanedVersion.yaml, md: cleanedVersion.md, id: req.params.id, title: article._doc.title }, res, req)
```